pkg/frontend/validate.go

```diff
@@ -118,3 +118,22 @@ func validateAdminVMName(vmName string) error {
 
 	return nil
 }
+
+func validateAdminKubernetesPodLogs(method, name, namespace, containerName string) error {
+	if method != http.MethodGet {
+		return api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodeInvalidParameter, "", "The provided HTTP method '%s' is invalid.", method)
+	}
+
+	if name == "" {
+		return api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodeInvalidParameter, "", "The provided pod name '%s' is invalid.", name)
+	}
+
+	if namespace == "" {
```

Comment:
This should be using the regex to validate the namespace as well.

Reply:
added the validation